buildSrc/src/main/groovy/io/micronaut/guides/GuideAsciidocGenerator.groovy

```diff
@@ -49,16 +52,16 @@ class GuideAsciidocGenerator {
     public static final String DEFAULT_APP_NAME = "default"
     private static final String COMMON_LICENSE = "common:license.adoc[]"
 
-    static void generate(GuideMetadata metadata, File inputDir,
+    static void generate(Guide metadata, File inputDir,
                          File asciidocDir, File projectDir) {
 
         JdkVersion javaVersion = Utils.parseJdkVersion()
-        if (metadata.maximumJavaVersion != null && javaVersion.majorVersion() > metadata.maximumJavaVersion) {
-            println "not generating asciidoc for $metadata.slug, JDK ${javaVersion.majorVersion()} > $metadata.maximumJavaVersion"
+        if (metadata.maximumJavaVersion( ) != null && javaVersion.majorVersion() > metadata.maximumJavaVersion( )) {
```

Comment:
```suggestion
        if (metadata.maximumJavaVersion() != null && javaVersion.majorVersion() > metadata.maximumJavaVersion()) {
```

Please don't use an empty space between parenthesis. We don't use that coding stye.